ColorPalette.js

```diff
@@ -5,12 +5,11 @@ define([
 	"./_TemplatedMixin",
 	"./_PaletteMixin",
 	"./hccss", // has("highcontrast")
-	"dojo/i18n", // i18n.getLocalization
```

Comment:
To address @wkeese's point about still needing to support non-AMD code, I'm thinking we could do this:

"dojo/has!csp-restrictions?dojo/i18n!dojo/nls/colors:dojo/i18n" // i18n or i18n.getLocalization